ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -54,62 +46,86 @@ const cromwellSupportArticles = [
     link: WORKFLOW_AND_WDL_LINK,
   },
 ];
-const DEFAULT_MACHINE_TYPE: Machine = findMachineByName(DEFAULT_MACHINE_NAME);
+
+const DEFAULT_MACHINE_TYPE: Machine = findMachineByName(
+  defaultCromwellConfig.kubernetesRuntimeConfig.machineType
+);
 
 const { cpu, memory } = DEFAULT_MACHINE_TYPE;
 
-const PanelMain = fp.flow(
-  withCdrVersions(),
-  withCurrentWorkspace(),
-  withUserProfile()
-)(
-  ({
-    analysisConfig,
-    workspace,
-    profileState,
-    creatorFreeCreditsRemaining,
-    onClose,
-  }) => {
-    // all apps besides Jupyter
-    const [userApps, setUserApps] = useState<UserAppEnvironment[]>();
-    const [creatingCromwellApp, setCreatingCromwellApp] = useState(false);
-
-    const app = findApp(userApps, UIAppType.CROMWELL);
-    const loadingApps = userApps === undefined;
-
-    useEffect(() => {
-      appsApi().listAppsInWorkspace(workspace.namespace).then(setUserApps);
-    }, []);
-
-    const onDismiss = () => {
-      onClose();
-      setTimeout(() => setSidebarActiveIconStore.next('apps'), 3000);
-    };
-
-    const onCreate = () => {
-      if (!creatingCromwellApp) {
-        setCreatingCromwellApp(true);
-        fetchWithErrorModal(
-          () => createUserApp(workspace.namespace, defaultCromwellConfig),
-          {
-            customErrorResponseFormatter: (error: ApiErrorResponse) =>
-              error?.originalResponse?.status === 409 && {
-                title: 'Error Creating Cromwell Environment',
-                message:
-                  'Please wait a few minutes and try to create your Cromwell Environment again.',
-                onDismiss,
-              },
-          }
-        ).then(() => onDismiss());
-      }
-    };
+const analysisConfig: Partial<AnalysisConfig> = {
+  machine: findMachineByName(
```

Comment:
Could this be replaced with:
`machine: DEFAULT_MACHINE_TYPE` ?

Reply:
That would be equivalent with the *current* `defaultCromwellConfig` but if we changed Cromwell's default configuration we would need to remember to update this code as well.

Going a level deeper, there is probably a way to use `DEFAULT_MACHINE_TYPE` in `defaultCromwellConfig` but that would require refactoring `defaultCromwellConfig` to a type other than `CreateAppRequest` (which does not accept a `Machine`). I think that's out of scope for this PR, but we could consider it when we start to make Cromwell configurable and that code becomes more relevant.

Reply:
Oh, wait - I didn't realize we were duplicating this logic. I'll update this in the next PR.

I had `DEFAULT_MACHINE_NAME` and `DEFAULT_MACHINE_TYPE` (from another file) confused.

Reply:
Update: This has been refactored in [this commit](https://github.com/all-of-us/workbench/pull/7600/commits/b1e1fd914a9b08e6944d5f6e5058be8ccf57adb4).